modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upload/DDMFormUploadFileEntryHandler.java

```diff
@@ -123,13 +121,10 @@ protected FileEntry addFileEntry(
 	private long _getDDMFormDefaultUserId(long companyId)
 		throws PortalException {
 
-		Company company = _companyLocalService.getCompany(companyId);
+		User user = _userLocalService.getUserByScreenName(
+			companyId, DDMFormConstants.DDM_FORM_DEFAULT_USER_SCREEN_NAME);
```

Comment:
@joaovictoralvesmartins can you change to use `getUserIdByScreenName`? Thanks!

Reply:
@carolmariaabb done!